docs/src/pages/getting-started/example-projects/example-projects.md

```diff
@@ -17,3 +17,15 @@ The source code for this documentation site is also included in the repository.
 This is a slightly more complex project.
 Check out the [`/docs`](https://github.com/mui-org/material-ui/tree/master/docs) folder for
 build instructions.
+
+## More advanced example projects
+
+Looking for a more advanced example project?
+
+If you want to start with a more complete and real world example, you may need to include a set of tools and packages that could take your application one step further and ready for production, you should consider:
```

Comment:
```suggestion
If you want to start with a more complete and real world example, you could take a look at:
```

I found the second clause verbose and a bit redundant - "more complete and real world" is succinct. It also made the sentence grammar a bit funky.